cmd/labs/project/installer_test.go

```diff
@@ -194,6 +195,12 @@ func TestInstallerWorksForReleases(t *testing.T) {
 
 	ctx := installerContext(t, server)
 
+	ctx, stub := process.WithStub(ctx)
+	stub.WithStdoutFor(`python[\S]+ --version`, "Python 3.10.5")
+	stub.WithStderrFor(`python[\S]+ -m venv .*/.databricks/labs/blueprint/state/venv`, "[mock venv create]")
+	stub.WithStderrFor(`python3 -m pip install .`, "[mock pip install]")
+	stub.WithStdoutFor(`python3 install.py`, "setting up important infrastructure")
```

Comment:
You're mixing regexes with replacing OS-specific path separators. I expect the Windows tests to fail because the class match for non-whitespace (`\S`) should be affected.

Reply:
Same for the `.`; it will be interpreted as any instead of the literal.

Reply:
let's value readability first